packages/FormElement/src/FormElement.js

```diff
@@ -71,85 +71,91 @@ function FormElement(props) {
     ...moreProps
   } = props;
 
-  const extratedChildren = extractChildren(children, [
+  const extractedChildren = extractChildren(children, [
     "FormElement.Description",
     "FormElement.Error",
     "FormElement.Help",
+    "FormElement.Instructions",
   ]);
   const ariaDescriptionId = React.useRef(uuidv4()).current;
-  const hasError = !!extratedChildren["FormElement.Error"] && !!extratedChildren["FormElement.Error"].props.children;
+  const ariaErrorId = React.useRef(uuidv4()).current;
+  const ariaInstructionsId = React.useRef(uuidv4()).current;
+  const hasError = !!extractedChildren["FormElement.Error"] && !!extractedChildren["FormElement.Error"].props.children;
   const uniqueInputId = React.useRef(uuidv4()).current;
   const inputId = isNil(id) || id === "" ? uniqueInputId : id;
-  let isFooterInserted = false;
 
   function renderFooter() {
-    if (isFooterInserted) return;
-
-    isFooterInserted = true;
-
     if (hasError) {
-      return extratedChildren["FormElement.Error"];
+      return React.cloneElement(extractedChildren["FormElement.Error"], {
+        ariaErrorId,
+      });
     }
 
-    if (extratedChildren["FormElement.Description"]) {
-      return React.cloneElement(extratedChildren["FormElement.Description"], {
+    if (extractedChildren["FormElement.Description"]) {
+      return React.cloneElement(extractedChildren["FormElement.Description"], {
         ariaDescriptionId,
       });
     }
 
     return null;
   }
 
+  function renderInstructions() {
+    if (extractedChildren["FormElement.Instructions"]) {
+      return React.cloneElement(extractedChildren["FormElement.Instructions"], {
+        ariaInstructionsId,
+      });
+    }
+  }
+
+  const nativeChildTypes = ["input", "textarea", "select"];
+
+  const renderFormElementChild = (child, index) => (
+    <div key={`child-key_${index}`} css={formElementChildStyle}>
```

Comment:
Ideally we wouldn't use `index` as the `key`.  The original code was referencing `child.key`, which is a good idea, but if it doesn't exist, maybe we can fallback on the `index`?  Or else maybe `uuidv4()`?

cc @alexzherdev

Reply:
Also, I still think it would be nice to include the `data-pka-anchor='form-element.child'` attribute here to give the consumer to ability to apply styling overrides cleanly if necessary.

Reply:
I tried using `uuidv4()`but i was losing input focus on every key press. Also child.key doesn't always exist as you say. I'll try and add some fallback logic.

Reply:
Good point about uuid.  I think we might have to require a `key` be included on children of `<FormElement>`.  We should add that to the README as part of the API. Can we possibly throw a warning if it's not included?

Reply:
`child.key` is auto-generated by React, should it be always there?

Reply:
In that case, yeah, let's just use `child.key`. Thanks, @allison-c!

Reply:
Oh i see the `extractChildren` function is adding the key

Reply:
I'm not a big fan of using the `data-pka-anchor` attributes for styling but thats for another discussion.

Reply:
I agree with you, _internally_ (within Paprika).  But once someone consumes it, they will need some kind of selector to apply styling to, otherwise you'll end up with gross stuff like `div > div:nth-child(3) > span`.